code/dqe/rdbhdbcompare.q

```diff
@@ -0,0 +1,10 @@
+\d .dqe
+hdbport:@[value;`hdbport;6003]
+rdbhdbcompare:{[tab]                                    // this function should be ran in the rdb
+  h:hopen (`::56003:admin:admin);
```

Comment:
even if line 2 was the right way to get the port in torq, you don't use it here :grin: 

I think you should be using built in TorQ stuff rather than hardcoding a port number - this is the whole reason discovery process exists :)

See this section of TorQ docs: http://aquaqanalytics.github.io/TorQ/conn/#retrieving-and-using-handles